src/js/post-911-gib-status/containers/Main.jsx

```diff
@@ -9,17 +11,28 @@ class Main extends React.Component {
   }
 
   render() {
+    let appContent;
+
+    if (this.props.available) {
+      appContent = this.props.children;
+    } else if (this.props.available === undefined) {
+      appContent = <LoadingIndicator message="Loading your application..."/>;
```

Comment:
I think we've been trying to make these messages more specific to what's being loaded.